crates/chainspec/src/spec.rs

```diff
@@ -607,6 +607,7 @@ impl From<Genesis> for ChainSpec {
                 hardforks.push((
                     EthereumHardfork::Paris.boxed(),
                     ForkCondition::TTD {
+                        activation_block_number: genesis.config.merge_netsplit_block.expect("Merge netsplit block is required"),
```

Comment:
@mattsse @Rjected, do I need to update `merge_netsplit_block` to not be optional in the genesis file?